app/views/content/life-as-a-teacher/chemistry/_article.html.erb

```diff
@@ -0,0 +1,127 @@
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching chemistry opens the door for your pupils to pursue exciting careers in fields as diverse as medicine and forensic science, to palaeontology and oceanography.</p>
+      <p>By solving challenging problems, your pupils could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you’ll guide your pupils though the skills they need to show their understanding of chemistry. You will teach them valuable life skills such as critical thinking, decision making and communication.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "For me, science is all about being inquisitive. I was always the kid in class who was thinking, 'I wonder how that works, or I wonder why that does that'. I recognise that so much in the kids I teach and being able to give them the tools to develop that curiosity and knowledge is genuinely exhilarating.",
+    name: "Holly, science teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">national curriculum for chemistry</a>.</p>
+    <p>You could have the opportunity to teach or support physics and biology lessons throughout key stages 3 and 4.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), you'll cover themes from the 3 science disciplines of chemistry, biology and physics.</p>
+    <p>The chemistry themes are:</p>
+    <ul>
+      <li>the particulate nature of matter</li>
+      <li>atoms, elements and compounds</li>
+      <li>pure and impure substances</li>
+      <li>chemical reactions</li>
+      <li>energetics</li>
+      <li>the periodic table</li>
+      <li>materials</li>
+      <li>earth and atmosphere</li>
+    </ul>
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) will be more specific to chemistry, and include:</p>
+    <ul>
+      <li>atomic structure and the periodic table</li>
+      <li>structure, bonding and the properties of matter</li>
+      <li>chemical changes</li>
+      <li>energy changes in chemistry</li>
+      <li>rate and extent of chemical change</li>
+      <li>chemical analysis</li>
+      <li>chemical and allied industries</li>
+      <li>earth and atmospheric science</li>
+    </ul>
```

Comment:
Shouldn't we include KS5? Lots of people are interested in teaching A level

Reply:
Checking with Fraser